### PR TITLE
add path to push codecov to 100%

### PR DESCRIPTION
**Pull request by zackt113**

### CODE REVIEW

**Comment by zackt113** [comment]
And I just added a minor change on my function for the axis name's spacing and format spacing

**Comment by voremargot** [comment]
The changes look great to me! Nice additions 👍

File: tests/testthat/test-plot_ci.R [hunk 1]
```diff
@@ -15,5 +15,5 @@ test_that("Check if error thrown for invalid path input value", {
 })
 
 test_that("Check the output is a ggplot object", {
-  expect_equal(is(plot_ci(c(1, 2, 3), 1000),"ggplot"), TRUE)
+  expect_equal(is(plot_ci(c(1, 2, 3), 1000, path = "."),"ggplot"), TRUE)
```
Comment (gauthampughazhendhi) [inline tests/testthat/test-plot_ci.R]:
```suggestion
  expect_equal(is(plot_ci(c(1, 2, 3), 1000, path = "."), "ggplot"), TRUE)
```

**Review by gauthampughazhendhi — APPROVED** [review]
LGTM

**Pull request merged**